.github/workflows/migration-check.yml

```
@@ -1,44 +1,69 @@
+name: Migration Safety Check
+
+"on":
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  migration-check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)